source/features/releases-tab.tsx

```diff
@@ -18,9 +18,9 @@ const repoUrl = getRepoURL();
 const repoKey = `releases-count:${repoUrl}`;
 
 // Get as soon as possible, to have it ready before the first paint
-const cached = cache.get(repoKey);
+const cached = cache.get<number>(repoKey);
 
-function updateReleasesCount(): number | Promise<number> {
+async function updateReleasesCount(): Promise<number | undefined> {
```

Comment:
I made this `async` so that we could just return a Promise since non-Promise values get wrapped as a resolved Promise.